spec/core.md

```diff
@@ -18,187 +18,253 @@ DID, after the prefix, is the case-sensitive [[ref: method-specific identifier]]
+verification of the KEL MAY provide mechanisms for detecting such behavior, such
+as KERI witnesses and watchers.
+
+### Handling Web Redirection
+
+A `did:webs` is intended to be a "stable" (long-lasting) identifier that can be
+put into documents, such as verifiable credentials, that are intended to be
+useful for a very long time -- generations. However, the web is not a very
+stable place, and documents are moved around and copied frequently. When two or
+more companies merge, often the web presence of some of the merged entities
+"disappears". It may not be possible to retain a permanent `did:webs` web
+location.
+
+When a `did:webs` moves, its AID does not change. The same KEL is used to verify
+the DIDDoc. Were the AID to change, it would be an altogether new DID,
+unconnected to the first DID. So if a resolver can find a newly named DID that
+uses the same AID, and the KEL verifies the DID, then they have resolved the
+DID. In fact, a `did:webs` could be moved to use another DID method that uses
+the AID for uniqueness and KEL for validity.
+
+The following are the capabilities in `did:webs` to help in the face of
+resolution uncertainty.
+
+- The `did:webs` is bound to its location via an event recording in the KEL, and
+  as the `id` in the DIDDoc.
+- When a `did:webs` is permanently moved to some other location, an event in the
+  KEL records the change.
+  - The `id` in the DIDDoc is set to the new location.
+  - An `AlsoKnownAs` entry is added to the DIDDoc for the old location.
+  - If possible, the controller of the DID MAY use web redirects to allow
+    resolution of the old location of the DID to the new location.
+
+If the previously published location of a `did:webs` is not redirected, an
+entity trying to resolve the DID MAY be able to find the data for the DID
+somewhere else using just the AID. Since the AID is globally unique and
+references the same identifier, regardless of the rest of the string that is the
+full `did:webs`, web searching could yield either the current location of the
+DIDDoc, or a copy of the DID that may be useful. For example, even the [Internet
+Archive: Wayback Machine](https://archive.org/web) could be used to find a copy
+of the DIDDoc and KEL at some point in the past, that may be sufficient for the
+purposes of the entity trying to resolve the DID. This specification does not
+rely on the Wayback Machine, but it might be a useful DID resolver tool.
+
+The DIDDoc, KEL and other files related to the DID may be cop
+
+### Identifiers in a `did:webs` DIDDoc
+
+Within the DIDDoc of a `did:webs` DID, the following rules are followed:
+
+- The `id` field in the root of the DIDDoc is the `did:webs` DID, reflecting the current combined web location and AID of the identifier.
+- The `alsoKnownAs` field in the root of the DIDDoc MAY contain other synonym resolvable `did:webs` DIDs. The `alsoKnownAs` field MAY contain `did:web` versions of the `did:webs` DID(s).
```

Comment:
I want to talk more about putting did:web in there. I'm not sure that a did:web and a did:webs are also known as one another. Rather, they both refer to the same subject, but they offer different semantics.

Reply:
I tend to agree on that.  Leave for now, but a point to be discussed.